src/python/detectors/pytorch_miss_call_to_eval/pytorch_miss_call_to_eval.py

```diff
@@ -0,0 +1,17 @@
+#  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#  SPDX-License-Identifier: Apache-2.0
+
+# {fact rule=pytorch-miss-call-to-eval@v1.0 defects=1}
```

Comment:
The code does not have any code that does evaluation, can you use the example below to adapt both cases? 

model.load_state_dict(torch.load("model.pth"))
classes = ["T-shirt/top","Trouser","Pullover","Dress","Coat","Sandal","Shirt","Sneaker","Bag","Ankle boot",]
model.eval()
x, y = test_data[0][0], test_data[0][1]
with torch.no_grad():
    pred = model(x)
    predicted, actual = classes[pred[0].argmax(0)], classes[y]
    print(f'Predicted: "{predicted}", Actual: "{actual}"')